qucs/components/digi_source.cpp

```diff
@@ -186,3 +188,83 @@ QString Digi_Source::verilogCode(int NumPorts)
   s += "  end\n";
   return s;
 }
+
+QString Digi_Source::spice_netlist(bool)
+{
+  QString s    = SpiceModel + Name;
+  QString port = spicecompat::normalize_node_name(Ports.at(0)->Connection->Name);
+  s += " " + port + " 0 "; // node names
+
+  QString V    = spicecompat::normalize_value(getProperty("V")->Value);
```

Comment:
It's need to check if PWL source allows using variables inside PWL function. If variables are not allowed, it would be sufficient to simple use `misc::str2num`

Reply:
PWL source allows using variables inside PWL function. But in this implementation of digital source it can not work due to the style of writing times in original QUCS digital source.